.travis.yml

```diff
@@ -13,11 +13,14 @@ matrix:
           python: 3.5
         - os: linux
           python: 3.6
-        - os: linux
-          python: 3.7-dev
+          env: WITH_COVERAGE=1
         - os: linux
           python: 3.6
-          env: PURE_PYTHON=1
+          env:
```

Comment:
It's not clear to me that we need entries in the matrix for C-extension-with-coverage and pure-python-with-coverage, since pure-python is the only thing that coverage can actually detect. I would imagine that the pure-python-with-coverage case would catch everything except for a tiny bit of code in `_import_c_module` and perhaps the  `_skip_` functions.

But hey, overall this whole PR only added just over a minute to the build time, so that's a very minor issue.

Reply:
See above.